recipes/demoji/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: demoji
+  version: {{ version }}
+
+source:
+  - folder: dist
+    url: https://pypi.io/packages/source/d/demoji/demoji-{{ version }}.tar.gz
+    sha256: 072efaeca725e6f63ab59d83abeb55b178842538ed9256455a82ebbd055ff216
+  - folder: src
+    url: https://github.com/bsolomon1124/demoji/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Can you use this one as the source as well to avoid two downloads?

Reply:
b047646c18